src/OpenTracing/Scope.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace OpenTracing;
+
+/**
+ * A {@link Scope} formalizes the activation and deactivation of a {@link Span}, usually from a CPU standpoint.
+ *
+ * <p>
```

Comment:
Where is this closed?
I would propose to use markdown in docblocks instead, not every IDE supports HTML and markdown is more readable when not rendered